lib/index.js

```diff
@@ -149,28 +160,37 @@ export default {
           linterName => configEnabledEngines.has(linterName));
 
         // Construct the command line invocation which runs the Code Climate CLI
-        const cmd = [this.executablePath, 'analyze',
-          '-f json',
-          makeEngineString(linterEnabledEngines),
-          `'${atom.project.relativize(filePath)}'`,
-          '< /dev/null'].join(' ');
+        const execArgs = ['analyze', '-f', 'json'].concat(
+          getEngineArgs(linterEnabledEngines),
+          [atom.project.relativize(filePath), '</dev/null']);
```

Comment:
Whoops, almost missed this. `project::relativize` isn't part of the public API, please switch this to using [`project::relativizePath`](https://atom.io/docs/api/v1.16.0/Project#instance-relativizePath). Note that you will need the first element from the array this returns, and you should handle the case where it returns `null` by falling back to the directory name of the current file.